docs/Changelog.md

```diff
@@ -15023,6 +15023,22 @@ This version of the operator has been available since version 13 of the default
   For example, a 64-bit float 3.1415926459 may be round to a 32-bit float 3.141592. Similarly, converting
   an integer 36 to Boolean may produce 1 because we truncate bits which can't be stored in the targeted type.
 
+  In more detail, the conversion among numerical types should follow these rules:
+
+  * Casting from float to:
+    * float: +/- infinity if OOR (out of range).
```

Comment:
It would be great to specify that it's about a case of casting to a lower-bit float, it may look confusing at first.

Reply:
I think @p-wysocki means replacing `float` by something like `{double, float, float16, bfloat16}` ... otherwise, a float-to-float conversion sounds confusing (if interpreted as float32).

Reply:
@gramalingam I see. How about we say "floating point" instead of "float". This should make it clear we are referring to all floating point formats.

Reply:
Fine with me. If so, we could use fixed point instead of int, uint.

Reply:
I changed them to "floating point" and "fixed point" as suggested.